### PR TITLE
[faq] Add tracks recording in background section troubleshooting

### PR DESCRIPTION
**Pull request by biodranik**
Please review and propose translations into your language.

### CODE REVIEW

File: content/faq/bookmarks/android-track-recording/index.es.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+taxonomies:
+faq: ["Marcadores y rutas"]
+
+extra:
+order: 30
+---
+
+La configuración predeterminada de optimización de la batería en Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC y otros dispositivos puede detener o cerrar la aplicación Organic Maps en segundo plano.
+
+Esto es especialmente cierto para las versiones modernas de Android:
```
Comment (andergd) [inline content/faq/bookmarks/android-track-recording/index.es.md:15]: The proposed sentence is not very common in Spanish. I would suggest:

"Esto ocurre especialmente en las versiones modernas de Android:"

File: content/faq/bookmarks/android-track-recording/index.es.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+- Android 14
+- Android 13
+- Android 12
+- Android 11
+
+Los pasos exactos sobre cómo hacer que Organic Maps (y otras aplicaciones) funcionen en segundo plano se enumeran aquí: https://dontkillmyapp.com/
```
Comment (andergd) [inline content/faq/bookmarks/android-track-recording/index.es.md:21]: I would suggest adding a note indicating that this webpage is in English before the colon: "(en inglés)"

File: content/faq/bookmarks/android-track-recording/index.uk.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+updated: "2024-10-05"
+
+taxonomies:
+  faq: ["Мітки та Маршрути"]
+
+extra:
+  order: 30
+---
+
+Стандартні налаштування оптимізації роботи акумулятора на Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC та інших пристроях можуть зупиняти або вбивати додаток Organic Maps у фоновому режимі.
```
Comment (strump) [inline content/faq/bookmarks/android-track-recording/index.uk.md:13]:
```suggestion
Стандартні налаштування оптимізації роботи акумулятора на пристроях Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC та інших пристроях можуть зупиняти або закривати додаток Organic Maps у фоновому режимі.
```

File: content/faq/bookmarks/android-track-recording/index.uk.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+---
+title: Чому треки (маршрути) не записуються у фоновому режимі на Android?
+description: "Стандартні налаштування оптимізації батареї на Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC та інших пристроях можуть зупинити або вбити додаток Organic Maps у фоновому режимі."
```
Comment (strump) [inline content/faq/bookmarks/android-track-recording/index.uk.md:3]:
```suggestion
description: "Стандартні налаштування оптимізації батареї на пристроях Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC та інших пристроях можуть зупинити або закрити додаток Organic Maps у фоновому режимі."
```

**Review by strump — COMMENTED** [review]
Fixes for Ukrainian translation:

**Comment by vng** [comment]
Just in case to link: https://github.com/organicmaps/organicmaps/issues/9319

File: content/faq/bookmarks/android-track-recording/index.pt-BR.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+updated: "2024-10-05"
+
+taxonomies:
+  faq: ["Favoritos e trilhas"]
+
+extra:
+  order: 30
+---
+
+As configurações padrão de otimização da bateria na Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC e outros dispositivos podem interromper ou eliminar o aplicativo Organic Maps em segundo plano.
```
Comment (matheusgomesms) [inline content/faq/bookmarks/android-track-recording/index.pt-BR.md:13]:
```suggestion
As configurações padrão de otimização da bateria na Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC e outros dispositivos podem interromper ou fechar o aplicativo Organic Maps em segundo plano.
```

File: content/faq/bookmarks/android-track-recording/index.pt-BR.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+Isso é especialmente verdadeiro para as versões modernas do Android:
```
Comment (matheusgomesms) [inline content/faq/bookmarks/android-track-recording/index.pt-BR.md:15]:
```suggestion
Isso acontece especialmente nas versões mais modernas do Android:
```

File: content/faq/bookmarks/android-track-recording/index.pt-BR.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+- Android 14
+- Android 13
+- Android 12
+- Android 11
+
+As etapas exatas sobre como fazer o Organic Maps (e outros aplicativos) funcionar em segundo plano estão listadas aqui: https://dontkillmyapp.com/
```
Comment (matheusgomesms) [inline content/faq/bookmarks/android-track-recording/index.pt-BR.md:21]:
```suggestion
O passo-a-passo sobre como fazer o Organic Maps (e outros aplicativos) funcionar em segundo plano pode ser encontrado aqui: https://dontkillmyapp.com/
```

File: content/faq/bookmarks/android-track-recording/index.pt.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
+---
+title: Porque é que as faixas não são gravadas de forma fiável em segundo plano no Android?
```
Comment (matheusgomesms) [inline content/faq/bookmarks/android-track-recording/index.pt.md:2]:
```suggestion
title: Porque é que os trajetos não são gravados de forma fiável em segundo plano no Android?
```

File: content/faq/bookmarks/android-track-recording/index.pt.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+description: As configurações padrão de otimização da bateria na Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC e outros dispositivos podem parar ou matar o aplicativo Organic Maps em segundo plano.
```
Comment (matheusgomesms) [inline content/faq/bookmarks/android-track-recording/index.pt.md:3]:
```suggestion
description: As definições predefinidas de otimização da bateria na Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC e outros dispositivos podem parar ou encerrar a aplicação Organic Maps em segundo plano.
```

File: content/faq/bookmarks/android-track-recording/index.pt.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+updated: "2024-10-05"
+
+taxonomies:
+  faq: ["Favoritos e trilhas"]
+
+extra:
+  order: 30
+---
+
+As definições predefinidas de otimização da bateria na Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC e outros dispositivos podem parar ou eliminar a aplicação Mapas orgânicos em segundo plano.
```
Comment (matheusgomesms) [inline content/faq/bookmarks/android-track-recording/index.pt.md:13]:
```suggestion
As definições predefinidas de otimização da bateria na Samsung, Huawei, Google, Xiaomi, OnePlus, Meizu, Asus, Wiko, Lenovo, Oppo, Vivo, Realme, Sony, Motorola, HTC e outros dispositivos podem parar ou encerrar a aplicação Organic Maps em segundo plano.
```

File: content/faq/bookmarks/android-track-recording/index.pt.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+
+Isso é especialmente verdadeiro para versões modernas do Android:
```
Comment (matheusgomesms) [inline content/faq/bookmarks/android-track-recording/index.pt.md:15]:
```suggestion
Isso acontece especialmente nas versões mais modernas do Android:
```

File: content/faq/bookmarks/android-track-recording/index.pt.md [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+- Android 14
+- Android 13
+- Android 12
+- Android 11
+
+Os passos exactos sobre como fazer com que os Mapas orgânicos (e outras aplicações) funcionem em segundo plano estão listados aqui: https://dontkillmyapp.com/
```
Comment (matheusgomesms) [inline content/faq/bookmarks/android-track-recording/index.pt.md:21]:
```suggestion
Os passos exatos sobre como fazer com que o Organic Maps (e outras aplicações) funcionem em segundo plano estão listados aqui: https://dontkillmyapp.com/
```

**Review by matheusgomesms — COMMENTED** [review]
PT and PT-BR reviewed

**Comment by biodranik** [comment]
Thanks for the translations!

**Pull request merged**